### ISSUE TITLE
[FEATURE] <Show questions>

### ISSUE THREAD

**Issue by ykdojo** [ykdojo/defaang#74]
### Description

Assuming we have #59, we should be able to show the questions on the UI.

**Comment by ShaileshParmar11** [comment]
do we have any rough mock for the same?

**Comment by ykdojo** [comment]
I don't think we do - unless we have something in #10

**Comment by ShaileshParmar11** [comment]
Let me know once some mocks come, I want to contribute!

**Comment by ykdojo** [comment]
Sure. I think something simple where you simply show all questions in a list will be ok at the beginning.

Also, [this page](https://tailwindui.com/components/application-ui/lists/stacked-lists) from [Tailwind UI](https://github.com/ykdojo/defaang/discussions/98) might be good for inspiration & code.

**Comment by ShaileshParmar11** [comment]
Thanks ill have a look!

**Comment by dattavelivela** [comment]
Hey, @ykdojo @ShaileshParmar11 

Can you let me know more about these Questions? I'll create a mockup as soon as possible in Figma.

Btw, I am Datta Velivela. An Aspiring UX Designer and UI Designer.

I can implement the Mock in Figma.

Thank you for your time and let me know about the mocks.

**Comment by ykdojo** [comment]
@dattavelivela thank you.

I believe there's a bit more info about the questions on #126 and #59.

**Comment by yaa-chi** [comment]
Hi @dattavelivela Thanks for joining in! Do have a look at what we have done here for the signup flow, we can use this Figjam to continue working on other mockups for the defaang website as we currently have a team file. I can add you to the team as well.

Here's the link: https://www.figma.com/file/DtaNEDhDNz0Wgs4XohV6Pt/Defaang-User-Flow?node-id=0%3A1

**Comment by dattavelivela** [comment]
Sure, please add me to the team file

Figma Acc- dattavelivela7@gmail.com

**Comment by ykdojo** [comment]
yes it is

**Comment by ashishcas** [comment]
hi @ykdojo  sorry i wasn't able to reply , can you assign it to me

**Comment by ashishcas** [comment]
@ykdojo  I am no able to find mockup its helpful if you can share the mock up I will start working

**Comment by ykdojo** [comment]
@ashishcas I don't think we have any mockup for it yet. Would you like to try making one?

**Comment by ashishcas** [comment]
I dont have much expertise in making mockups , but will try that

**Comment by ykdojo** [comment]
Inspiration #1: reddit

![Image](https://user-images.githubusercontent.com/1811651/196756736-df54c228-c37c-4e14-983c-9c4c6169d303.png)

**Comment by Akshay1018** [comment]
@ykdojo the above design seems good. is any design for the Show question/ how it works finalized or not? This will be really interesting if we start working on it. What are your thoughts on this @ykdojo?

**Comment by yaa-chi** [comment]
> Sure, please add me to the team file. And I'll take a look into the Figjam file by eod.
> 
> Figma Acc- [dattavelivela7@gmail.com](mailto:dattavelivela7@gmail.com)

Done⚡
